force-app/main/default/objects/Session_Registration__c/Session_Registration__c.object-meta.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@bmtemakina let's create 2 additional lookup fields on Session registration object, one is for Contact and the other one is for public event. 

Because during session registration creation user has to select an event registration record which is displayed like "E-{0000}", for a user that doesn't mean anything and it's difficult for a user to find the correct Event Registration record.

We could pre-populate public event lookup and populate event registration lookup post creation.